<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试客户端</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>417</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0,2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTextEdit" name="Receive_textEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_middle">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="IP_address_label">
        <property name="text">
         <string>服务端IP地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IP_address_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="Port_label">
        <property name="text">
         <string>服务端端口号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Port_lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="Connect_pushButton">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Disconnect_pushButton">
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_bottom">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QTextEdit" name="Send_textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="Send_pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
